salt/roster/dir.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+'''
+Read in the rosters in a directory using the renderer system.
+'''
+from __future__ import absolute_import
+
+# Import python libs
+import os
+from fnmatch import fnmatch
+import re
+import copy
+import logging
+
+import salt.loader
+import salt.config
+from salt.template import compile_template
```

Comment:
same here, this should just be `import salt.template` and then `salt.template.compile_template` should be used so that it can just be seen immediately what is being used in the function